saas/models/saas_template.py

```diff
@@ -14,15 +14,18 @@
 _logger = logging.getLogger(__name__)
 
 MANDATORY_MODULES = ['auth_quick']
-DEFAULT_PYTHON_CODE = """# Available variables:
+DEFAULT_BUILD_PYTHON_CODE = """# Available variables:
 #  - env: Odoo Environment on which the action is triggered
 #  - model: Odoo Model of the record on which the action is triggered; is a void recordset
 #  - record: record on which the action is triggered; may be void
 #  - records: recordset of all records on which the action is triggered in multi-mode; may be void
 #  - time, datetime, dateutil, timezone: useful Python libraries
 #  - log: log(message, level='info'): logging function to record debug information in ir.logging table
 #  - Warning: Warning Exception to use with raise
-# To return an action, assign: action = {{...}}\n\n\n\n"""
+# To return an action, assign: action = {{...}}
+# You can specify places for variables that can be passed when creating a build like this:
+# env['{{key_name_1}}'].create({{'subject': '{{key_name_2}}', }})
```

Comment:
Why it's double brackets here? It mislead user. What about removing comments from code before formating?